extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsoleProcessor.java

```diff
@@ -123,7 +123,10 @@
 
     private static final Logger log = Logger.getLogger(DevConsoleProcessor.class);
 
-    private static final String STATIC_RESOURCES_PATH = "dev-static/";
+    private static final GACT DEVCONSOLE_WEBJAR_ARTIFACT_KEY = new GACT("io.quarkus", "quarkus-vertx-http-deployment", null,
+            "jar");
+    private static final String DEVCONSOLE_WEBJAR_STATIC_RESOURCES_PATH = "dev-static/";
+    private static final String DEVCONSOLE_FINAL_DESTINATION = "META-INF/dev-ui-files";
```

Comment:
Why do we need this now? Couldn't it be generated from the artifact id/version + resource path? That would simplify things a bit.
At least I don't think we should make it mandatory.

Reply:
I removed this option completly. Automaticly generating the final destination also prevents clashes between different webjars, which is quite nice.